qiskit/visualization/qcstyle.py

```diff
@@ -47,32 +47,33 @@ def __init__(self):
         self.sfs = 8
         self.disptex = {
             'id': 'I',
-            'u0': 'U_0',
-            'u1': 'U_1',
-            'u2': 'U_2',
-            'u3': 'U_3',
+            'u0': '$\\mathrm{U}_0$',
+            'u1': '$\\mathrm{U}_1$',
+            'u2': '$\\mathrm{U}_2$',
+            'u3': '$\\mathrm{U}_3$',
             'x': 'X',
             'y': 'Y',
             'z': 'Z',
             'h': 'H',
             's': 'S',
-            'sdg': 'S^\\dagger',
+            'sdg': '$\\mathrm{S}^\\dagger$',
             't': 'T',
-            'tdg': 'T^\\dagger',
+            'tdg': '$\\mathrm{T}^\\dagger$',
             'iswap': 'Iswap',
             'dcx': 'Dcx',
             'ms': 'MS',
             'diagonal': 'Diagonal',
             'unitary': 'Unitary',
             'r': 'R',
-            'rx': 'R_x',
-            'ry': 'R_y',
-            'rz': 'R_z',
-            'rxx': 'R_{xx}',
-            'ryy': 'R_{yy}',
-            'rzx': 'R_{zx}',
-            'reset': '\\left|0\\right\\rangle',
-            'initialize': '|psi>'
+            'rx': '$\\mathrm{R}_\\mathrm{X}$',
+            'ry': '$\\mathrm{R}_\\mathrm{Y}$',
+            'rz': '$\\mathrm{R}_\\mathrm{Z}$',
+            'rxx': '$\\mathrm{R}_{\\mathrm{XX}}$',
+            'ryy': '$\\mathrm{R}_{\\mathrm{YY}}$',
+            'rzx': '$\\mathrm{R}_{\\mathrm{ZX}}$',
+            'rzz': '$\\mathrm{R}_{\\mathrm{ZZ}}$',
+            'reset': '$\\left|0\\right\\rangle$',
+            'initialize': '$|psi>$'
```

Comment:
```suggestion
            'initialize': '$|\\psi\\rangle$'
```

Reply:
I don't think this `\\psi` works with the default font. I get a 'Glyph missing from current font' message. If there's interest in replacing the psi with the Greek symbol, it can be done in unicode in python. Do we want cap psi or the italic lower psi?

Reply:
`$\\psi\\rangle$` works fine on my system and I haven't changed the default fonts in mpl.

![Screenshot_2020-07-20_07-35-25](https://user-images.githubusercontent.com/2447371/87933647-a490a900-ca5b-11ea-8a93-f60764a13dff.png)

It is probably dependent on your system fonts, and is not a matplotlib issue.